test/unit/mdc-list/mdc-list.test.js

```diff
@@ -273,22 +273,6 @@ test('wrapFocus calls setWrapFocus on foundation', () => {
   td.verify(mockFoundation.setWrapFocus(true), {times: 1});
 });
 
-
-test('singleSelection true sets the click handler from the root element', () => {
-  const {root, component, mockFoundation} = setupTest();
-  component.singleSelection = true;
-  domEvents.emit(root, 'click');
-  td.verify(mockFoundation.handleClick(td.matchers.anything()), {times: 1});
-});
-
-test('singleSelection false removes the click handler from the root element', () => {
```

Comment:
FYI there is no equivalent test to this in the new code (i.e. no test that tests MDCListFoundation#handleClick when `isSingleSelectionList_` is false)